biometric-login/test/configs/wdio.ios.legacy.rdc.conf.js

```diff
@@ -0,0 +1,63 @@
+const { config } = require('./wdio.shared.sauce.conf');
+const testName = `iOS Biometric login real device Legacy RDC: ${new Date().getTime()}`;
+
+// =================
+// Service Providers
+// =================
+// These are not needed and need to be removed, else the test will fail. We want to re-use properties
+delete config.user;
+delete config.key;
+
+// ============
+// Capabilities
+// ============
+// For all capabilities please check
+// http://appium.io/docs/en/writing-running-appium/caps/#general-capabilities
+config.capabilities = [
+  {
+    // Just give an iPhone with FaceID
+    deviceName: 'iPhone X*',
+    // The api key that has a reference to the app-project in the TO cloud
+    testobject_api_key: process.env.SAUCE_RDC_EU_ACCESS_KEY_IOS,
+    // The name of the test for in the cloud
+    testobject_test_name: testName,
+    // Some default settings
+    // You can find more info in the TO Appium Basic Setup section
+    platformName: 'iOS',
+    idleTimeout: 180,
+    testobject_cache_device: true,
+    noReset: true,
+    orientation: 'PORTRAIT',
+    newCommandTimeout: 180,
+    // Always default the language to a language you prefer so you know the app language is always as expected
+    language: 'en',
+    locale: 'en',
+    autoAcceptAlerts:true,
+    // Enable touchID on RDC
+    allowTouchIdEnroll: true,
+  },
+  {
+    // Just give an iPhone with TouchId
+    deviceName: 'iPhone [678]S',
```

Comment:
Should the S have a ? after it, indicating that it could be the 6, 7, or 8, or the 6S, 7S, or 8S? I have an iPhone 6 (not a 6S), and it has TouchId. Also, should there be a `\+?`, indicating that it could be a 6S+, 7S+, or 8S+

Reply:
You're right, I'll change it. thanks